.github/workflows/release.yaml

```diff
@@ -53,4 +53,23 @@ jobs:
       - uses: cucumber-actions/publish-npm@v1.0.0
         with:
           npm-token: ${{ secrets.NPM_TOKEN }}
-          working-directory: 'javascript'
\ No newline at end of file
+          working-directory: javascript
+
+  publish-java:
+    name: Publish Maven package
+    needs: create-release
+    runs-on: ubuntu-latest
+    environment: Release
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+
+      - uses: samuelmeuli/action-maven-publish@v1
```

Comment:
At least it's simple, but this plugin amounts to little more than [one shell command](https://github.com/samuelmeuli/action-maven-publish/blob/master/index.js#L84). I'd prefer to roll our own for the reasons stated above.

Reply:
We also need to import a GPG key, but yeah - rolling our own is fine. Total ownership, simplicity and more secure.

Reply:
👍